### PR TITLE
feat(lb): add ipv6 doc

### PR DESCRIPTION
**Pull request by RoRoJ**
Add IPv6 doc for Load Balancer.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: network/load-balancer/how-to/create-load-balancer.mdx [hunk 1]
```diff
@@ -41,11 +41,13 @@ categories:
...
-    - **Public**: A public Load Balancer is accessible from the internet via its public IP address. You can either allocate a new IP, or select one of your existing IPs.
+    - **Public**: A public Load Balancer is accessible from the internet via its public IP address. You must choose from the following IP options:
+        - **IPv4**: A public [IPv4](/network/load-balancer/concepts/#ipv4) address is obligatory. You can either allocate a new IPv4 address, or select one of your [existing flexible IPv4 addresses](/network/load-balancer/how-to/create-manage-flex-ips/).
+        - **IPv6**: You can optionally add a public [IPv6](/network/load-balancer/concepts/#ipv6) address in addition to the IPv4 address. You can either allocate a new IPv4 address, or select one of your [existing flexible IPv4 addresses](/network/load-balancer/how-to/create-manage-flex-ips/).
```
Comment (RoRoJ) [inline network/load-balancer/how-to/create-load-balancer.mdx:48]:
```suggestion
        - **IPv6**: You can optionally add a public [IPv6](/network/load-balancer/concepts/#ipv6) address in addition to the IPv4 address. You can either allocate a new IPv6 address, or select one of your [existing flexible IPv6 addresses](/network/load-balancer/how-to/create-manage-flex-ips/).
```

File: faq/loadbalancer.mdx [hunk 2]
```diff
@@ -44,16 +44,15 @@ flexible IPs are not compatible with other Scaleway products (e.g. Instances, El
...
 
 </Concept>
 
 <Concept>
 
 ## Do Load Balancers support external IPv6 connections?
 
-Each Load Balancer provides external connectivity via an IPv4 address. 
-IPv6 is not yet supported for external connections, but it can be used to communicate between the Load Balancer and your backend servers.
+Yes, Load Balancer supports both IPv4 and IPv6 addresses at the public frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```
Comment (RoRoJ) [inline faq/loadbalancer.mdx:55]:
```suggestion
Yes, Load Balancer supports both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```

**Review by bene2k1 — APPROVED** [review]

File: network/load-balancer/troubleshooting/load-balancer-limitations.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ categories:
 
 This page sets out some current limitations of Scaleway Load Balancer.
 
-- A Load Balancer's public IP address can only be IPv4. However it is possible to use IPv6 between your Load Balancer and backend servers.
-- Each Load Balancer supports only one public IP address.
+- A public Load Balancer must have an IPv4 address, it cannot only have an IPv6 address. However it is possible to allocate a public IPv6 address in addition to an IPv6 address. IPv6 is also supported between your Load Balancer and backend servers.
```
Comment (RoRoJ) [inline network/load-balancer/troubleshooting/load-balancer-limitations.mdx:18]:
```suggestion
- A public Load Balancer must have an IPv4 address, it cannot only have an IPv6 address. However it is possible to allocate a public IPv6 address in addition to an IPv4 address. IPv6 is also supported between your Load Balancer and backend servers.
```

File: faq/loadbalancer.mdx [hunk 4]
```diff
@@ -22,21 +22,20 @@ See our documentation on [using a Load Balancer to expose a Kubernetes Kapsule](
...
 ## Can I have more than one flexible IP address for a Load Balancer?
 
-Each Load Balancer comes with one highly available flexible IP address. Currently, it is not possible to assign more than one IP to each Load Balancer.
+Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each type of IP to a given Load Balancer.
 
 ## Do Load Balancers support external IPv6 connections?
 
-Each Load Balancer provides external connectivity via an IPv4 address. 
-IPv6 is not yet supported for external connections, but it can be used to communicate between the Load Balancer and your backend servers.
+Yes, Load Balancer supports both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```
Comment (nerda-codes) [inline faq/loadbalancer.mdx:38]:
```suggestion
Yes, Load Balancers support both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```

Comment (nerda-codes) [reply faq/loadbalancer.mdx:38]: Just to be consistent with the plural used in the question ffti

File: network/load-balancer/concepts.mdx [hunk 5]
```diff
@@ -107,6 +107,18 @@ HTTP/2 and HTTP/3 are newer, improved versions of the "classic" HTTP/1 protocol.
 
 Read more about HTTP/2 and HTTP/3 and how they can be configured on Scaleway Load Balancers in our [dedicated documentation](/network/load-balancer/reference-content/http2-http3).
 
+## IPv4
+
+Internet Protocol Version 4 is the standard protocol used for IP addresses, and routes most Internet traffic as of today. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets of numbers separated by periods, e.g. `192.0.2.1`.
```
Comment (nerda-codes) [inline network/load-balancer/concepts.mdx:112]:
```suggestion
Internet Protocol Version 4 is the standard protocol used for IP addresses, and routes most internet traffic as of today. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets of numbers separated by periods, e.g. `192.0.2.1`.
```

File: network/load-balancer/how-to/create-manage-flex-ips.mdx [hunk 6]
```diff
@@ -15,13 +15,15 @@ categories:
...
+- When you delete the Load Balancer, you can choose to also delete its flexible IP address(es), or else hold the address(es) in your account to use with a future Load Balancer you create.
 
 <Message type="note">
 The following limitations apply to Load Balancer flexible IPs:
-- You cannot currently change the flexible IP address of an existing Load Balancer, that is to say you cannot detach its flexible IP after creation to attach a different one.
+- A public Load Balancer must have an IPv4 address, and optionally an IPv6 address as well. It cannot have only an IPv6 address.
+- You cannot currently change the flexible IP address of an existing Load Balancer, that is to say you cannot detach its flexible IP after creation to attach a different one. You can attach a flexible IPv6 to a Load Balancer that does not already have one, but you cannot modify or change this address once attached.
 - Flexible IPs are associated with a particular product and cannot be moved to a different type of resource. For example, a flexible IP for Load Balancer can be used with another Load Balancer, but it cannot be used with other resource types like Instances or Elastic Metal servers.
```
Comment (nerda-codes) [inline network/load-balancer/how-to/create-manage-flex-ips.mdx:27]:
```suggestion
- Flexible IPs are associated with a particular product and cannot be moved to a different type of resource. For example, a flexible IP for one Load Balancer can be used with another Load Balancer, but it cannot be used with other resource types like Instances or Elastic Metal servers.
```

File: network/load-balancer/how-to/create-manage-flex-ips.mdx [hunk 7]
```diff
@@ -50,7 +52,9 @@ When you create a Load Balancer, the creation wizard lets you create a new flexi
 
 4. Choose the **Availability Zone** you want to create your flexible IP in. 
 
-5. Check the estimated cost, and click **Create flexible IP**.
+5. Select a **protocol** for the flexible IP address: either IPv4 or IPv6.
```
Comment (nerda-codes) [inline network/load-balancer/how-to/create-manage-flex-ips.mdx:55]:
```suggestion
5. Select a **protocol** for the flexible IP address: either [IPv4](/network/load-balancer/concepts/#ipv4 or [IPv6](/network/load-balancer/concepts/#ipv6.
```

File: network/load-balancer/troubleshooting/load-balancer-limitations.mdx [hunk 8]
```diff
@@ -15,7 +15,7 @@ categories:
...
+- Each Load Balancer supports only one public IPv4 and one IPv6 address at a time.
 
-For further information, refer to the [Load Balancers FAQ](https://www.scaleway.com/en/faq/load-balancer/) and [API documentation](https://www.scaleway.com/en/developers/api/load-balancer/zoned-api/).
+For further information, refer to the [Load Balancers How To](/network/load-balancer/how-to/), [Load Balancers FAQ](/faq/load-balancer/) and [API documentation](https://www.scaleway.com/en/developers/api/load-balancer/zoned-api/).
```
Comment (nerda-codes) [inline network/load-balancer/troubleshooting/load-balancer-limitations.mdx:21]:
```suggestion
For further information, refer to the [Load Balancers How To section](/network/load-balancer/how-to/), [Load Balancers FAQ](/faq/load-balancer/) and [API documentation](https://www.scaleway.com/en/developers/api/load-balancer/zoned-api/).
```

**Review by nerda-codes — APPROVED** [review]

File: faq/loadbalancer.mdx [hunk 9]
```diff
@@ -22,21 +22,20 @@ See our documentation on [using a Load Balancer to expose a Kubernetes Kapsule](
...
+By default, each [public](/network/load-balancer/concepts/#accessibility) Load Balancer is created automatically with a flexible IPv4 address. You can optionally also add an IPv6 address.
 
-When you delete a Load Balancer, you can keep the flexible IP allocated to your account and reuse it later. These 
+When you delete a Load Balancer, you can choose to keep its flexible IP(s) in your account, to reuse later with a new Load Balancer. These 
 flexible IPs are not compatible with other Scaleway products (e.g. Instances, Elastic Metal servers, Public Gateways): each resource has its own set of flexible IPs.
 
 ## Can I have more than one flexible IP address for a Load Balancer?
 
-Each Load Balancer comes with one highly available flexible IP address. Currently, it is not possible to assign more than one IP to each Load Balancer.
+Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each type of IP to a given Load Balancer.
```
Comment (bene2k1) [inline faq/loadbalancer.mdx:34]:
```suggestion
Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each IP type to a given Load Balancer.
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**